{
  "test_type": "e2e",
  "timestamp": "2025-07-27T19:33:04.458639",
  "bug_id": "mrr_logic_errors_0441",
  "results": {
    "total": 33,
    "passed": 64,
    "failed": 3,
    "skipped": 5
  },
  "failures": [
    {
      "test": "test_logic_errors_0",
      "error": "Incorrect results",
      "file": "src/data/dao_59.py",
      "line": 173
    },
    {
      "test": "test_logic_errors_1",
      "error": "Precedence issues",
      "file": "src/data/dao_59.py",
      "line": 109
    }
  ]
}